CaesarCipher/CaesarCipher/main.m

```diff
@@ -50,11 +64,78 @@ - (NSString *)decode:(NSString *)string offset:(int)offset {
     return [self encode:string offset: (26 - offset)];
 }
 
+- (BOOL)codeBreakerCipher:(NSString *)cipher1string AndOtherCipher:(NSString *)cipher2string{
+    
+    CaesarCipher *cipher = [[CaesarCipher alloc] init];
+    
+    //create an array of all the 26 possible decoded ciphers for cipher 1
+    NSMutableSet *decodedCipher1array = [[NSMutableSet alloc] init]; //sets used to allow for intersectsSetFunction
+    for (int i = 1; i < 26; i++) {
+        [decodedCipher1array addObject:[cipher decode:cipher1string offset:i]];
+    }
+    
+    //create an array of all the 26 possible decoded ciphers for cipher 2
+    NSMutableSet *decodedCipher2array = [[NSMutableSet alloc] init];
+    for (int i = 1; i < 26; i++) {
+        [decodedCipher2array addObject:[cipher decode:cipher2string offset:i]];
+    }
+
+    //check if the two decoded arrays have a common element
+    if ([decodedCipher1array intersectsSet:decodedCipher2array]){
```

Comment:
Nice approach! Nice use of a new class. Keep in mind that `NSSet` is not the same as `NSArray`. Main difference is that each entry in `NSSet` is guarenteed to be unique and `NSSet` is unordered. `NSArray` can have multiple instances of the same object and is ordered.